build_from.py

```diff
@@ -18,31 +19,36 @@
 logger.setLevel(logging.INFO)
 logger.addHandler(logging.StreamHandler())
 
-
 if __name__ == "__main__":
-    if len(sys.argv) < 6:
+    if len(sys.argv) < 5:
         print("Usage: python build_from.py regulation.xml title "
               + "notice_doc_# act_title act_section (Generate diffs? "
```

Comment:
I think this line needs to change, because the notice_doc_# is no longer needed as a parameter right?